pkg/envoy/server.go

```diff
@@ -63,7 +63,8 @@ var (
 		// Allow all TCP traffic to any port.
 		{Protocol: envoy_config_core.SocketAddress_TCP},
 		// Allow all UDP traffic to any port.
-		{Protocol: envoy_config_core.SocketAddress_UDP},
+		// UDP rules not sent to Envoy for now.
+		// {Protocol: envoy_config_core.SocketAddress_UDP},
```

Comment:
I assume we don't have any users using UDP for envoy and this won't break existing installations?

Reply:
Right. This will change in some future with QUIC/HTTP3.